example/lib/main.dart

```diff
@@ -265,7 +280,7 @@ class _ContentState extends State<Content> {
                 },
                 title: 'Title text',
                 description:
-                    'This text is just for test and we dont care about it at all.',
+                    Text('This text is just for test and we dont care about it at all.'),
```

Comment:
"don't*", missed that I guess we care a bit 🤣 
```suggestion
                    Text('This text is just for test and we don't care about it at all.'),
```